package/root/usr/local/sbin/rock64_health.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+set -e
+
+if [ "$(id -u)" -ne "0" ]; then
+	echo "This script requires root."
+	exit 1
+fi
+
+print() {
+	printf "%-15s: %s %s\n" "$1" "$2 $3" "$4"
+}
+
+cpu_frequency() {
+	local cur=$(cat /sys/devices/system/cpu/cpu0/cpufreq/cpuinfo_cur_freq)
+	local mhz=$(awk "BEGIN {printf \"%.2f\",$cur/1000}")
+	print "CPU freq" $mhz "MHz"
+}
+
+scaling_govenor() {
+	local gov=$(cat /sys/devices/system/cpu/cpu0/cpufreq/scaling_governor)
+  	print "Governor" $gov
+}
+
+cpu_count() {
+	local cpus=$(grep -c processor /proc/cpuinfo)
+	print "CPU count" $cpus
+}
+
+soc_temp() {
+	local RAW_TEMP=$(cat /sys/class/thermal/thermal_zone0/temp)
+	local TEMP=$(awk "BEGIN {printf \"%.2f\",$RAW_TEMP/1000}")
+	print "SoC Temp" $TEMP "C"
+}
+
+usage() {
+	echo "Usage: $0 [-w] [-h]"
+}
+
+all() {
+	cpu_frequency
+	cpu_count
+	scaling_govenor
+	soc_temp
+}
+
+
+WATCH=""
+for i in "$@"; do
+	case $i in
```

Comment:
I do not intend to change it. This is simply a minor modification of an existing script from the pine64, not a re-write.